.github/workflows/package.yml

```diff
@@ -2,99 +2,79 @@ name: Package
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
 
 env:
   CARGO_TERM_COLOR: always
   cargo-deb-version: 1.28.0
 
 jobs:
-  build-armv7:
+  build:
     runs-on: ubuntu-latest
-    env:
-      target: armv7-unknown-linux-gnueabihf
-      binutils: binutils-arm-linux-gnueabihf
+    strategy:
+      matrix:
+        target:
+          - armv7-unknown-linux-gnueabihf
+          - aarch64-unknown-linux-gnu
+          - # Empty target to build for host architecture (x86_64-unknown-linux-gnu) without cross.
+        include:
+          - target: armv7-unknown-linux-gnueabihf
+            binutils: binutils-arm-linux-gnueabihf
+          - target: aarch64-unknown-linux-gnu
+            binutils: binutils-aarch64-linux-gnu
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Install dependencies
-      run: sudo apt-get install libdbus-1-dev ${{ env.binutils }}
-    - name: Toolchain
-      uses: actions-rs/toolchain@v1
-      with:
-        toolchain: stable
-        target: ${{ env.target }}
-        override: true
-
-    - name: Cache .cargo and target
-      uses: actions/cache@v2
-      with:
-        path: |
-          ~/.cargo/git
-          ~/.cargo/registry
-          target
-        key: ${{ runner.os }}-package-cargo-${{ hashFiles('**/Cargo.lock') }}
-    - name: Cache cargo-deb and cross
-      uses: actions/cache@v2
-      with:
-        path: |
-          ~/.cargo/bin/cargo-deb
-          ~/.cargo/bin/cross
-          ~/.cargo/.crates.toml
-          ~/.cargo/.crates2.json
-        key: ${{ runner.os }}-cargo-bin-${{ env.cargo-deb-version }}
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: sudo apt-get install libdbus-1-dev ${{ matrix.binutils }}
+      - name: Toolchain
+        if: matrix.target != ''
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: ${{ matrix.target }}
+          override: true
 
-    - name: Install cargo-deb
-      run: cargo install cargo-deb --version ${{ env.cargo-deb-version }}
+      - name: Cache .cargo and target
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/git
+            ~/.cargo/registry
+            target
+          key: ${{ runner.os }}-package-cargo-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
also matrix.target to reduce the download size? (How big is it before/after this change, anyway?)

Reply:
Done. On some recent run it was 184 MB.